components/automate-gateway/gateway/services.go

```diff
@@ -874,12 +874,15 @@ func (s *Server) authRequest(r *http.Request, resource, action string) (context.
 
 	projects := auth_context.ProjectsFromMetadata(md)
 
-	authzResp, err := s.authorizer.IsAuthorized(ctx, subjects, resource, action, projects)
+	newCtx, authorized, err := s.authorizer.IsAuthorized(ctx, subjects, resource, action, projects)
 	if err != nil {
+		// any error is handled as "permission denied" at the call sites, with the error text returned
```

Comment:
When I read this, I think "ok, so if there's an error, it's unauthorized, but also, it's unauthorized if the bool `authorized ` is false . . . 

Are we really saying "an err is returned if authorization can't be determined. Upstream services, however, will consider it equivalent to an explicit permission denied, with error message included"? (words up for refinement, just trying to get the meaning)